src/pro/sky/java/course1/Employee.java

```diff
@@ -0,0 +1,53 @@
+package pro.sky.java.course1;
+
+public class Employee {
+
+    static int counter = 1;
+    private final String name;
```

Comment:
визуально лучше отделять статические поля от нестатических
```suggestion
    private static int counter = 1;
    
    private final String name;
```